### PR TITLE
Feature/nex 81 es2015 migration

### PR DESCRIPTION
**Pull request by krampstudio**
Related to https://oat-sa.atlassian.net/browse/NEX-81

 - Bundles with babel
 - migrate some controllers to ES2015 (partially)

Requires PR : 

 - [x] https://github.com/oat-sa/tao-core/pull/2156

### CODE REVIEW

File: views/js/controller/main.js [hunk 1]
```diff
@@ -32,56 +32,48 @@ function ($, taskQueueManagerFactory, taskQueue) {
...
+            .hide();//start hidden to prevent blinking effect
 
             //listen to events started by the task queue model
-            taskQueue.on('taskcreated', function(data){
-                if(taskManager.list.is('hidden')){
-                    taskManager.absorbBurst(data.sourceDom, [0, 300, 600]).then(function(){
-                        taskManager.addNewTask(data.task);
+            taskQueue
+                .on('taskcreated', data => {
+                    if(taskManager.list.is('hidden')){
```
Comment (ferenckiss89) [inline views/js/controller/main.js]: Sometimes we put spaces between IF and the open parenthesis, sometimes not. Maybe if we already update these "smaller" files we could also fix the format issues.

**Review by ferenckiss89 — APPROVED** [review]
A small comment on format but ok for me.

File: views/js/controller/main.js [hunk 2]
```diff
@@ -32,56 +32,48 @@ function ($, taskQueueManagerFactory, taskQueue) {
      * @exports taoTaskQueue/controller/main
      */
     return {
-        start: function () {
+        start() {
 
             var taskManager = taskQueueManagerFactory({
```
Comment (btamas) [inline views/js/controller/main.js]: It stayed `var`

**Review by ferenckiss89 — APPROVED** [review]
Ok for me.

**Review by anshul119 — APPROVED** [review]

**Review by btamas — APPROVED** [review]

**Pull request merged**